python/tox.ini

```diff
@@ -23,7 +23,7 @@ usedevelop = true
 deps =
     coverage
     mock
-    pytest
+extras = dev
```

Comment:
Could this be done in a separate PR?

Reply:
This is needed so it will import other `dev` dependencies automatically. 
With `extras=dev`, we can remove `pytest` because it is already included in `dev.